node-group-project/src/components/PetTypeContainer.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
+import { Link } from "react-router-dom";
 
 const PetTypeContainer = (props) => {
-  return (
+  const [adoptablePets, setAdoptablePets] = useState([]);
+
+  let petType = props.match.params.petType;
+
+  useEffect(() => {
+    fetch(`http://localhost:3000/pets/${petType}`)
+      .then(result => setAdoptablePets(result.rows))
+      .catch(error => console.log(error))
+  }, []);
```

Comment:
This might be a good place to use petType as an anchor value. It'll become necessary if you want them to be able to switch pet types on the page. For now though, this looks great!